src/views/home/home.js

```diff
@@ -29,26 +32,12 @@ export default {
   },
   methods: {
     filterPodcasts(term) {
-      if (!term) {
-        this.podcasts = [...this.initialPodcasts];
-        return;
-      }
-
-      this.podcasts = this.initialPodcasts.filter((podcast) => {
-        const track = podcast.trackName.toLowerCase();
-        return track.includes(term.toLowerCase());
-      });
+      const termSanitized = term.toLowerCase().trim();
+      this.podcastListFiltered = filterBySearchTerm(termSanitized, this.podcasts);
     },
-    filterFavorites(term) {
-      if (!term) {
-        this.favorites = [...this.initialFavorites];
-        return;
-      }
-
-      this.favorites = this.initialFavorites.filter((favorite) => {
-        const track = favorite.trackName.toLowerCase();
-        return track.includes(term.toLowerCase());
-      });
+    async filterFavorites(term) {
+      const termSanitized = term.toLowerCase().trim();
```

Comment:
We already have a function for this into our utils (trim) Either move it to that function or create a new one to wrap up lowercase will be my suggestion

Reply:
Added to filterBySearchTerm function